structures/lists.py

```diff
@@ -20,25 +29,40 @@ def last_of_four_digits(numbers: []) -> []:
     original numbers, preserving the initial order.
     for example [1004, 9181, 1700, 4565] -> [4, 1, 0, 5]
     :param numbers: the original list of four digit integers
-    :type number: []
+    :type numbers: []
     :returns: a new list of last digits only
     :rtype: []
     """
     # TODO implement last_of_four_digits()
-    return []
+    last = []
+    i = 0
+    while i < len(numbers):
+        last.append(numbers[i] % 10)
```

Comment:
-3 no comments, expecting a description of *why* numbers[i] % 10 is an acceptable solution